### PR TITLE
Adjust travis.yml for Go tool moves.

### PR DESCRIPTION
**Pull request by kisom**
c.f. https://golang.org/doc/go1.5#moving

### CODE REVIEW

**Comment by jkroll-cf** [comment]
LGTM.

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,8 @@ language: go
 go:
   - tip
 before_script:
-  - go get golang.org/x/tools/cmd/vet
   - go get golang.org/x/tools/cmd/goimports
   - go get -u github.com/golang/lint/golint
-  - go get golang.org/x/tools/cmd/cover
```
Comment (jkroll-cf) [inline .travis.yml]: Were we actually using cover?

Comment (kisom) [inline .travis.yml]: I don't think so yet; I'm fairly sure this was pulled from the CFSSL travis.yml.

Comment (jacobhaven) [inline .travis.yml]: Confirmed.

**Pull request merged**